pkg/cmd/roachtest/roachtestutil/mixedversion/testdata/planner/shared_process

```diff
@@ -1,7 +1,7 @@
 # Test that we are able to generate test plans for tests using a
 # shared-process deployment mode.
 
-mixed-version-test predecessors=(21.2.11, 22.1.8, 22.2.3, 23.1.4, 23.2.0) num_upgrades=5 deployment_mode=shared-process
+mixed-version-test predecessors=(21.2.11, 22.1.8, 22.2.3, 23.1.28, 23.2.0) num_upgrades=5 deployment_mode=shared-process
```

Comment:
This one changed because we decoupled `maybeClampMsbMsv` from the predecessor, whereas the test overrides only `predecessorFunc`, so the patch is still be clamped inside `chooseUpgradePath`.